wallet/src/de/schildbach/wallet/ui/invite/InvitationCreatedFragment.kt

```diff
@@ -51,15 +62,20 @@ class InvitationCreatedFragment : Fragment(R.layout.fragment_invitation_created)
         appCompatActivity.setSupportActionBar(toolbar)
 
 //        profile_picture_envelope.avatarProfile = requireArguments().getParcelable()
-        diaplayOwnProfilePicture() // just for the testing purposes
+        displayOwnProfilePicture() // just for the testing purposes
         preview_button.setOnClickListener {
             showPreviewDialog()
         }
         send_button.setOnClickListener {
-            val errorDialog = FancyAlertDialog.newInstance(R.string.invitation_creating_error_title,
-                    R.string.invitation_creating_error_message, R.drawable.ic_error_creating_invitation,
-                    R.string.okay, 0)
-            errorDialog.show(childFragmentManager, null)
+//            val errorDialog = FancyAlertDialog.newInstance(R.string.invitation_creating_error_title,
+//                    R.string.invitation_creating_error_message, R.drawable.ic_error_creating_invitation,
+//                    R.string.okay, 0)
+//            errorDialog.show(childFragmentManager, null)
```

Comment:
Remove if not needed.

Reply:
Let's leave it for now, it can be useful in https://dashpay.atlassian.net/browse/NMA-261

Reply:
ok, I will see if it is.  There were some errors that I got about insufficient funds in the logs, but there was no dialog for it!